src/parser/jobs/gnb/modules/Tincture.tsx

```diff
@@ -0,0 +1,30 @@
+import {Trans} from '@lingui/react'
+import {SEVERITY} from 'parser/core/modules/Suggestions'
+import {Tincture as CoreTincture} from 'parser/core/modules/Tincture'
+import React from 'react'
+
+export default class Tincture extends CoreTincture {
+	buffAction = this.data.actions.INFUSION_STR
+
+	trackedActions = {
+		icon: this.data.actions.INFUSION_STR.icon,
+		actions: [
+			{
+				action: this.data.actions.WICKED_TALON,
```

Comment:
So if a user pots between Savage Claw and Wicket Talon, or early pot and it falls off after Wicked Talon without catching Eye Gouge, that's ok? I'm not entirely sure how this presents but might want to include the full continuation combo, maybe also DoTs; basically everything that goes into No Mercy at a minimum.

Reply:
Requires modifications to core, to be addressed in https://github.com/xivanalysis/xivanalysis/issues/1158.

Reply:
DoTs are ignored for the same reason No Mercy is, namely the cooldown not lining up with 4:30 pots, the rest addressed in linked issue.